samples/kubernetes-event-sources/k8s-events-sa.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
```

Comment:
the file says `-sa.yaml` but isn't the actual `kind: ServiceAccount` file missin' in this `ClusterRoleBinding` ?